Uplift/Views/SportsDetail/SportsDetailPlayersCollectionViewCell.swift

```diff
@@ -0,0 +1,117 @@
+    private let caratImage = UIImageView()
+    private let divider = UIView()
+    private let headerLabel = UILabel()
+    private let previewLabel = UILabel()
+    private let playersTextView = UITextView()
+    
+    private var players: [User] = []
+    private var dropStatus: DropdownStatus = .closed
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    func setupViews() {
+        headerLabel.font = ._16MontserratBold
+        headerLabel.textAlignment = .center
+        headerLabel.numberOfLines = 0
+        headerLabel.textColor = .primaryBlack
+        addSubview(headerLabel)
+        
+        caratImage.image = UIImage(named: ImageNames.rightArrowSolid)
+        caratImage.contentMode = .scaleAspectFit
+        caratImage.clipsToBounds = true
+        addSubview(caratImage)
+        
+        previewLabel.font = ._14MontserratLight
+        previewLabel.textAlignment = .center
+        previewLabel.numberOfLines = 0
+        previewLabel.textColor = .primaryBlack
+        addSubview(previewLabel)
+        
+        playersTextView.isEditable = false
+        playersTextView.textAlignment = .center
+        playersTextView.font = ._14MontserratLight
+        playersTextView.textColor = .primaryBlack
+        playersTextView.isScrollEnabled = false
+        addSubview(playersTextView)
+        
+        divider.backgroundColor = .gray01
+        addSubview(divider)
+        
+    }
+    
+    func setupConstraints() {
+        let headerCaratHorizontalPadding = 12
+        let headerPlayersVerticalPadding = 14
+        let horizontalPadding = 40
+        let verticalPadding = 24
+        headerLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalToSuperview().offset(verticalPadding)
+        }
+        
+        caratImage.snp.makeConstraints { make in
+            make.centerY.equalTo(headerLabel)
+            make.height.equalTo(10)
+            make.width.equalTo(10)
+            make.leading.equalTo(headerLabel.snp.trailing).offset(headerCaratHorizontalPadding)
+        }
+        
+        previewLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(horizontalPadding)
+            make.bottom.equalToSuperview().inset(verticalPadding)
+        }
+        
+        playersTextView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(horizontalPadding)
+            make.bottom.equalToSuperview().inset(verticalPadding)
+            make.top.equalTo(headerLabel.snp.bottom).offset(headerPlayersVerticalPadding)
+        }
+        
+        divider.snp.makeConstraints { make in
+            make.bottom.leading.trailing.equalToSuperview()
+            make.height.equalTo(1)
+        }
+    }
+    
+    func configure(for post: Post, dropStatus: DropdownStatus) {
+        headerLabel.text = "PLAYERS \(post.players)/10"
```

Comment:
Client string

Reply:
need post object

Reply:
Can we put just the "PLAYERS" part as a client string?